### PR TITLE
Update Cert Manager Section in Self-Hosted Docs

### PR DESCRIPTION
**Pull request by jimmybrancaccio**
## Description

Adds a small bit about `cert-manager` being able to also fetch TLS certificates from Let's Encrypt which can be used on your instance.

## Related Issue(s)

N/A

## How to test

N/A

## Release Notes

N/A

## Documentation

N/A

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2121"></a>

### CODE REVIEW

File: gitpod/docs/self-hosted/latest/required-components.md [hunk 1]
```diff
@@ -70,4 +70,4 @@ Gitpod expects to be connected to a Source Control Management System (SCM) such
 
 ## Cert Manager
 
-[Cert-manager](https://cert-manager.io/) must be installed in your cluster before you can install Gitpod. It is used to create and manage certificates needed to secure communication between the various internal Gitpod components. See the [getting started guide](./getting-started#step-2-install-cert-manager) for details on how to install cert-manager.
+[Cert-manager](https://cert-manager.io/) must be installed in your cluster before you can install Gitpod. It is used to create and manage certificates needed to secure communication between the various internal Gitpod components. It can even be used to fetch a TLS certificate for your Gitpod instance from Let's Encrypt for example. See the [getting started guide](./getting-started#step-2-install-cert-manager) for details on how to install cert-manager.
```
Comment (lucasvaltl) [inline gitpod/docs/self-hosted/latest/required-components.md:73]: I suggest rewording this from "even be used" to "also be used". This makes it seem a bit more formal.

**Review by lucasvaltl — APPROVED** [review]
lgtm - thanks for adding this! :) I only suggested a small wording change.

File: gitpod/docs/self-hosted/latest/required-components.md (hunk 1 above)
Comment (mikenikles) [inline gitpod/docs/self-hosted/latest/required-components.md:73]:
```suggestion
[Cert-manager](https://cert-manager.io/) must be installed in your cluster before you can install Gitpod. It is used to create and manage certificates needed to secure communication between the various internal Gitpod components. It can also be used to fetch a TLS certificate for your Gitpod instance from Let's Encrypt for example. See the [getting started guide](./getting-started#step-2-install-cert-manager) for details on how to install cert-manager.
```
